Tests/Functional/config/bc_automatic_controller_injections.yml

```diff
@@ -0,0 +1,12 @@
+imports:
+    - { resource: default.yml }
+    
+jms_di_extra:
+    automatic_controller_injections:
+        properties:
+            context: "@security.context"
```

Comment:
This is used only when available. There is no need to bump the minimum SF version, we can make the bundle compatible with sf 3.0 as is. 
The build fails because of a dependency.